src/content/docs/network-performance-monitoring/advanced/snmp-profiles.mdx

```diff
@@ -4,108 +4,2980 @@ tags:
+              MAX-ACCESS accessible-for-notify
+              STATUS     obsolete
+              DESCRIPTION
+                  "This is the name of the VM in the notification."
+              ::= { vmwVC 303 }
+          
+          vmwVpxdOldStatus OBJECT-TYPE
+              SYNTAX     SnmpAdminString
+              MAX-ACCESS accessible-for-notify
+              STATUS     current
+              DESCRIPTION
+          	"This is the old status in the notification."
+              ::= { vmwVC 304 }
+          
+          vmwVpxdNewStatus OBJECT-TYPE
+              SYNTAX     SnmpAdminString
+              MAX-ACCESS accessible-for-notify
+              STATUS     current
+              DESCRIPTION
+          	"This is the new status in the notification."
+              ::= { vmwVC 305 }
+          
+          vmwVpxdObjValue OBJECT-TYPE
+              SYNTAX     VmwLongSnmpAdminString
+              MAX-ACCESS accessible-for-notify
+              STATUS     current
+              DESCRIPTION
+          	"This is the current object value in the notification."
+              ::= { vmwVC 306 }
+          
+          vmwVpxdTargetObj OBJECT-TYPE
+              SYNTAX     SnmpAdminString
+              MAX-ACCESS accessible-for-notify
+              STATUS     current
+              DESCRIPTION
+          	"This is the current object in the notification. This may be
+          	one of esx host name, vm name, or other. This value must not be empty."
+              ::= { vmwVC 307 }
+          
+          vmwVpxdTargetObjType OBJECT-TYPE
+              SYNTAX   INTEGER { unknown(1), host(2), vm(3), other(4)}
+              MAX-ACCESS accessible-for-notify
+              STATUS     current
+              DESCRIPTION
+          	"This is the alarm target object type."
+              ::= { vmwVC 308 }
+          ```
+
+          This snippet contains 2 major sections:
+
+          * **Notifications by an agent that implements this mib module** - This is the definition of an actual trap message, identified by a unique OID.
+            * [vmwVCNotifications](https://www.circitor.fr/Mibs/Html/V/VMWARE-VC-EVENT-MIB.php#vmwVCNotifications) - This is the root notification OID which will cover the 3 defined types of messages available: [vpxdAlarm](https://www.circitor.fr/Mibs/Html/V/VMWARE-VC-EVENT-MIB.php#vpxdAlarm), [vpxdDiagnostic](https://www.circitor.fr/Mibs/Html/V/VMWARE-VC-EVENT-MIB.php#vpxdDiagnostic), and [vpxdAlarmInfo](https://www.circitor.fr/Mibs/Html/V/VMWARE-VC-EVENT-MIB.php#vpxdAlarmInfo).
+          * **Notification parameters** - Commonly referred to as 'variable bindings' or 'varbinds', these are the unique key:value pairs of alert data that the SNMP agent can possibly send as decorations on the trap messages.
+            * In this MIB, there are 8 possible varbinds; all referenced as the OIDs ranging from `1.3.6.1.4.1.6876.4.3.301` - `1.3.6.1.4.1.6876.4.3.308`. This will be unique to every vendor and MIB.
```

Comment:
```suggestion
            * In this MIB, there are eight possible varbinds&mdash;all referenced as the OIDs ranging from `1.3.6.1.4.1.6876.4.3.301` - `1.3.6.1.4.1.6876.4.3.308`. This will be unique to every vendor and MIB.
```